join-chat-to-teams-meeting/Chat Experience with File Sharing/client.js

```diff
@@ -0,0 +1,217 @@
+import {
+    CallClient,
+    CallAgent
+} from "@azure/communication-calling";
+import {
+    AzureCommunicationTokenCredential
+} from "@azure/communication-common";
+import {
+    CommunicationIdentityClient
+} from "@azure/communication-identity";
+import {
+    ChatClient
+} from "@azure/communication-chat";
+
+let call;
+let callAgent;
+let chatClient;
+let chatThreadClient;
+
+const meetingLinkInput = document.getElementById('teams-link-input');
+const threadIdInput = document.getElementById('thread-id-input');
+const callButton = document.getElementById("join-meeting-button");
+const hangUpButton = document.getElementById("hang-up-button");
+const callStateElement = document.getElementById('call-state');
+
+const messagesContainer = document.getElementById("messages-container");
+const chatBox = document.getElementById("chat-box");
+const sendMessageButton = document.getElementById("send-message");
+const messagebox = document.getElementById("message-box");
+
+var userId = '';
+var messages = '';
+var tokenString = '';
+
+async function init() {
+    const connectionString = "<SECRET_CONNECTION_STRING>";
+    const endpointUrl = "<ENDPOINT_URL>";
+
+    const identityClient = new CommunicationIdentityClient(connectionString);
+
+    let identityResponse = await identityClient.createUser();
+    userId = identityResponse.communicationUserId;
+    console.log(`\nCreated an identity with ID: ${identityResponse.communicationUserId}`);
+
+    let tokenResponse = await identityClient.getToken(identityResponse, [
+        "voip",
+        "chat",
+    ]);
+
+    const {
+        token,
+        expiresOn
+    } = tokenResponse;
+    tokenString = token;
+    console.log(`\nIssued an access token that expires at: ${expiresOn}`);
+    console.log(token);
+
+    const callClient = new CallClient();
+    const tokenCredential = new AzureCommunicationTokenCredential(token);
+    callAgent = await callClient.createCallAgent(tokenCredential);
+    callButton.disabled = false;
+
+    chatClient = new ChatClient(
+        endpointUrl,
+        new AzureCommunicationTokenCredential(token)
+    );
+
+    console.log('Azure Communication Chat client created!');
+}
+
+init();
+
+callButton.addEventListener("click", async () => {
+    // join with meeting link
+    call = callAgent.join({
+        meetingLink: meetingLinkInput.value
+    }, {});
+
+    call.on('stateChanged', () => {
+        callStateElement.innerText = call.state;
+    })
+    // toggle button and chat box states
+    chatBox.style.display = "block";
+    hangUpButton.disabled = false;
+    callButton.disabled = true;
+
+    messagesContainer.innerHTML = messages;
+
+    console.log(call);
```

Comment:
Do we need this console.log here?

Reply:
not sure why it was added in the first place, this sample is based on original sample https://github.com/Azure-Samples/communication-services-javascript-quickstarts/tree/johnp/file-sharing/join-chat-to-teams-meeting/Chat%20Experience and it was added back in 2021